### PR TITLE
Add database insertion

### PR DESCRIPTION
**Pull request by jackjohn7**
- Can now insert data from the scrape into the database
- Fetching subjects for the `apitest` route

Use `npx prisma studio` to view the database rows (it's really nice!)

### CODE REVIEW

File: WebScraper/seleniumNavigator.py [hunk 1]
```diff
@@ -92,19 +93,17 @@ def run_scraper(term, output_to_json: bool = False, num_fails: int = 0):
             print("Max number of fails reached! Exiting...")
     else:
         print("Finished Successfully!")
-    finally:
```
Comment (CamJak) [inline WebScraper/seleniumNavigator.py:95]: Why was this removed? I know its not necessary but it was nice for still getting output when errors occurred.

**Review by CamJak — COMMENTED** [review]
Other than that it looks good to me! We're super close to a real prototype.

File: WebScraper/seleniumNavigator.py (hunk 1 above)
Comment (jackjohn7) [reply WebScraper/seleniumNavigator.py:95]: Gonna add the FInally clause back. It won't work the same though. I removed it because I didn't like that it would output json even it it failed or was exited prematurely. Now, the finally case should only run if there are errors or interrupts

Comment (CamJak) [reply WebScraper/seleniumNavigator.py:95]: Works for me.

**Review by CamJak — DISMISSED** [review]

**Review by CamJak — APPROVED** [review]
